src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
-import './App.scss';
+import React, { useEffect, useState } from 'react';
 import { MoviesList } from './components/MoviesList';
+import { Movie } from './types/Movie';
+
 import moviesFromServer from './api/movies.json';
+import './App.scss';
+
+const singleCaseQuery = (query: string) => {
+  return query.toLowerCase();
+};
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState<Movie[]>([]);
+
+  useEffect(() => {
+    setVisibleMovies(moviesFromServer);
+  }, []);
```

Comment:
You could just put the array into using `useState`

Reply:
I know, but I think will be better to upload less information at the start for a faster page render